{{ cookiecutter.app_name }}/pyproject.toml

```diff
@@ -39,11 +39,18 @@ requires = [
 ]
 system_requires = [
 {%- if cookiecutter.gui_framework == 'Toga' %}
-    'libgirepository1.0-dev',
+    'gir1.2-webkit-3.0',
     'libcairo2-dev',
+    'libgirepository1.0-dev',
+    'libgtk-3-dev',
     'libpango1.0-dev',
+    'librsvg2-dev',
     'libwebkitgtk-3.0-0',
```

Comment:
According to the [linuxdeploy-plugin-gtk README](https://github.com/linuxdeploy/linuxdeploy-plugin-gtk/blob/master/README.md), most or all of these libraries are already installed by the plugin. In that case, I think they should be removed from system_requires, both to avoid confusion about where the libraries are coming from, and to simplify the pyproject file.

Reply:
Sorry, disregard, I understand now. The plugin copies libraries from the build environment into the AppImage, but Briefcase is still responsible for making sure those libraries are installed in the build environment to begin with.